var/spack/repos/builtin/packages/hicops/package.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hicops(Package):
+    """HiCOPS is a software framework for accelerating database peptide search
+    workflows on supercomputers. HiCOPS provided algorithm-independent
+    parallelizations and optimizations can be extended into new HPC database search
+    algorithms or scalably accelerate the existing ones.
+    """
+
+    homepage = "https://hicops.github.io/index"
+    git = "https://github.com/hicops/hicops.git"
+    maintainers = ['pcdslab', 'mhaseeb123', 'nessiecancode']
+
+    version('release', branch='release')
+    version('develop', branch='develop')
+
+    # Build Options
+    variant('USE_MPI', default='ON',
+            description='CMake Option: Enable MPI support.',
+            values=('ON', 'OFF'))
+    variant('USE_TIMEMORY', default='OFF',
+            description='CMake Option: Enable timemory interface. Requires timemory '
+                        'installation.',
+            values=('ON', 'OFF'))
+    variant('USE_MPIP_LIBRARY', default='OFF',
+            description='CMake Option: Enables the MPIP data_tracker via Timemory. '
+                        'Requires timemory installation.',
+            values=('ON', 'OFF'))
+    variant('TAILFIT', default='ON',
+            description='CMake Option: Use the tailfit method instead of Gumbelfit '
+                        'for e-value computation.',
+            values=('ON', 'OFF'))
+    variant('PROGRESS', default='ON',
+            description='CMake Option: Display HiCOPS progress marks.',
+            values=('ON', 'OFF'))
+    variant('MAX_SEQ_LEN', default='60',
+            description='CMake Option: Allowed maximum peptide sequence length.')
+    variant('QALEN', default='100',
+            description='CMake Option: Maximum number of top K peaks to keep when '
+                        'spectrum preprocess.')
+    variant('QCHUNK', default='10000',
+            description='CMake Option: Max size of each batch extracted from the '
+                        'dataset.')
+    variant('MAX_HYPERSCORE', default='100',
+            description='CMake Option: Maximum allowed hyperscore computed.')
+
+    # Build Controls
+    variant('CMAKE_BUILD_TYPE', default='RelWithDebInfo',
+            description='CMake Build Control: Build type.',
+            values=('Release', 'Debug', 'MinSizeRel', 'RelWithDebInfo'))
```

Comment:
There is no need to set this if you inherit from `CMakePackage`

Reply:
It has been removed.